cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -32,6 +32,7 @@ bEnableEnhancedDeterminism=True # default is False
 bSubstepping=True               # default is False
 MaxSubstepDeltaTime=0.015625    # default is 0.016667 (16 substeps * 0.015625 seconds per substep == 0.25 seconds)
 MaxSubsteps=16                  # default is 6        (16 is the maximum allowed in the editor)
+SolverOptions=(PositionIterations=16,VelocityIterations=2,ProjectionIterations=2,CollisionMarginFraction=0.050000,CollisionMarginMax=10.000000,CollisionCullDistance=3.000000,CollisionMaxPushOutVelocity=1000.000000,ClusterConnectionFactor=1.000000,ClusterUnionConnectionType=DelaunayTriangulation,bGenerateCollisionData=False,CollisionFilterSettings=(FilterEnabled=False,MinMass=0.000000,MinSpeed=0.000000,MinImpulse=0.000000),bGenerateBreakData=False,BreakingFilterSettings=(FilterEnabled=False,MinMass=0.000000,MinSpeed=0.000000,MinVolume=0.000000),bGenerateTrailingData=False,TrailingFilterSettings=(FilterEnabled=False,MinMass=0.000000,MinSpeed=0.000000,MinVolume=0.000000))
```

Comment:
Do we know for certain that these settings add value and do not detract excessively from the frame rate?

Reply:
Ah ok. Maybe I thought I needed updated settings to achieve smoother URDF robot behavior. I'm not sure if this is necessary, but your demo might be relying on these settings somehow.

Reply:
Yeah, I just checked and my demo does rely on these settings.